tools/accuracy_checker/accuracy_checker/data_readers/data_reader.py

```diff
@@ -180,8 +179,94 @@ def read(self, data_id):
 class ScipyImageReader(BaseReader):
     __provider__ = 'scipy_imread'
```

Comment:
I think a note should be made in the documentation that this doesn't literally use SciPy anymore.